devtools/test_dashboard/index-mathjax3.html

```diff
@@ -24,6 +24,10 @@ <h4>no MathJax: Apple: $2, Orange: $3</h4>
     <div id="graph"></div>
   </div>
   <div id="snapshot"></div>
+  <div style="position:absolute; border-top: .3em solid #eee; bottom: 0px; left: 0; width: 100%">
```

Comment:
I'm not sure if we want this to be included in every test dashboard by default. Maybe we could add a special flag for that?